src/cycleclock.zig

```diff
@@ -1,23 +1,23 @@
 const builtin = @import("builtin");
 const std = @import("std");
 
-pub inline fn cycletime() u64 {
+pub inline fn now() u64 {
     const cpu: std.Target.Cpu = builtin.cpu;
-    if (comptime cpu.arch.isAARCH64()) {
-        return aarch64_cycletime();
+    const os: std.Target.Os = builtin.os;
+
+    if (comptime os.tag.isDarwin()) {
```

Comment:
it's hard to crosscompile with checks like these but we can worry about that later. There was some chatter in zig discord about having runtime os value available